src/deployment.ts

```diff
@@ -49,6 +49,12 @@ export interface DeploymentProps extends workload.WorkloadProps {
    * @default Duration.seconds(600)
    */
   readonly progressDeadline?: Duration;
+  /**
+   * Number of desired replicasets history.
```

Comment:
If it makes sense, we just copy the kubernetes docs. I think it makes sense here.

```suggestion
   * Specify how many old ReplicaSets for this Deployment you want to retain. The rest will be garbage-collected in the background. By default, it is 10.
```